### DISCUSSION TITLE
How could I set default size for floating window?

### DISCUSSION THREAD

**Discussion by Jedsek** [zellij-org/zellij#1332] [Q&A]
See title.

**Comment by imsnif** [comment]
Unfortunately there's no way to do this yet before opening the window. After opening the floating pane you can resize it as you would a normal pane (eg. with `alt +/-`).

**Reply by Jedsek** [reply]
@imsnif I think should be easy to solve, are there any difficulty?

**Reply by imsnif** [reply]
The difficulty with such issues is usually not technical, but rather about working hands.

**Comment by Jedsek** [comment]
Related issue: #1335

**Comment by Rianico** [comment]
I set the default size for a floating window through the layout:
```kdl
layout {
    pane {
        pane
    }
    floating_panes {
        pane {
            x "10%"
            width "81%"
            height "50%"
        }
    }
}
```
Now I can open a floating window with the customized size even though have closed it. But the downside is that each Zellij session will start with a floating window for the first time.

**Reply by imsnif** [reply]
If you do this with the default [swap layout](https://zellij.dev/documentation/swap-layouts) instead, it won't be opened on every session and you'll get the same effect.

**Comment by guttermonk** [comment]
This worked for me:
```
    keybinds {
        normal {
            bind "Super l" {
                Run "lazygit" {
                    close_on_exit true
                    floating true
                    x "10%"
                    y "15%"
                    width "80%"
                    height "70%"
                }
            }
        }
    }
```
